### DISCUSSION TITLE
How can I provide feedback to Github's help?

### DISCUSSION THREAD

**Discussion by berndbausch** [community/community#21915] [New to GitHub]
<p>I would like to improve a specific help page. Is there a place I can send feedback to?</p>
<p>In case anybody in charge is reading this: The page in question is <a href="https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/#platform-linux" rel="nofollow noopener">https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/#platform-linux</a>, and I have just spent some time trying to get Jekyll installed on Fedora 23 (step 2-5 on that page).</p>
<p>In the process, I found that I needed a few more packages, namely <em>ruby-devel</em>, <em>autoconf</em>, <em>automake</em> and <em>libtool</em>. It would be great if these requirements were mentioned on the help page.</p>
<p>It would also be a great time saver if the page explained what to do if I have existing Github pages. There is a paragraph starting with “If you want to use an existing Jekyll site repository…”, but I don’t know whether my Github pages count as “an existing Jekyll site repository” or not. For the moment, I am still experimenting.</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/berndbausch">@berndbausch</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and our docs. I’ve logged your feedback in our internal feedback list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Thanks again!</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi again <a class="mention" href="/berndbausch">@berndbausch</a>!</p>
<p>After connecting with the appropriate team, I had some follow up for you on this.</p>
<p>The packages you mentioned <em>shouldn’t</em> be required, as long as you’re using the github-pages gem. Can you clarify which OS and version you’re installing on? Alternatively, if you’d like to have a more in-depth conversation, you can write us at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> and we can dive a little deeper.</p>
<p>Thanks!</p>

**Comment by berndbausch** [comment]
<p>Thanks much, Nadia. I use Fedora 23, not the newest Fedora, but up to date.</p>
<p>Unfortunately I didn’t keep the error messages, but before I installed all the above, I ran into fatal errors when following the instructions. </p>
<p>By now I am able to run Jekyll and don’t need help (that statement is actually wrong, since I continue to feel like a <a href="http://www.spacechimps.com/theirstory.html" rel="nofollow noopener">monkey operating a spaceship</a> when dealing with Ruby:).</p>
